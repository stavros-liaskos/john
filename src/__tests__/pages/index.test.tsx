import React from 'react';
import { render } from '@testing-library/react';
import Home from 'pages/index';

describe('Index (Home)', () => {
  it('renders a heading', () => {
    const component = render(<Home />);

    const main = component.queryAllByRole('main');

    expect(main.length).toBe(1);
    expect(component).toMatchSnapshot();
  });
});
