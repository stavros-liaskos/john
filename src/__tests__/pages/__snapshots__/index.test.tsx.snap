// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index (Home) renders a heading 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container max-auto"
      >
        <div
          class="flex flex-col h-screen justify-between"
        >
          <main
            class="mb-auto"
          >
            <button>
              <svg
                class="h-6 w-6"
                fill="none"
                height="40"
                stroke="currentColor"
                viewBox="0 0 24 24"
                width="40"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                />
              </svg>
            </button>
            <div
              class="flex justify-center items-center h-48 bg-red-300"
            >
              <form
                class="flex justify-center items-stretch h-10"
                novalidate=""
              >
                <input
                  class="rounded mr-4 px-2 max-w-md"
                  name="search"
                  placeholder="Here goes the artist..."
                  type="text"
                />
                <button
                  class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
                  type="submit"
                >
                  Search
                </button>
              </form>
            </div>
            <div
              class="border border-indigo-100"
            >
              <div
                class="flex justify-between items-center px-8 even:bg-gray-100"
              >
                <p>
                  Led Zeppeling
                </p>
                <div
                  class="flex"
                >
                  <a
                    class="inline"
                    href="www.google.com"
                  >
                    <svg
                      height="40"
                      viewBox="0 0 128 128"
                      width="40"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <g
                        clip-rule="evenodd"
                      >
                        <path
                          d="M0 0h128v128H0z"
                          fill="none"
                        />
                        <path
                          d="M64 0C28.8 0 0 28.8 0 64s28.8 64 64 64 64-28.8 64-64S99.52 0 64 0m29.44 92.48c-1.28 1.92-3.52 2.56-5.44 1.28-15.04-9.28-33.92-11.2-56.32-6.08-2.24.64-4.16-.96-4.8-2.88-.64-2.24.96-4.16 2.88-4.8 24.32-5.44 45.44-3.2 62.08 7.04 2.24.96 2.56 3.52 1.6 5.44m7.68-17.6c-1.6 2.24-4.48 3.2-6.72 1.6-17.28-10.56-43.52-13.76-63.68-7.36-2.56.64-5.44-.64-6.08-3.2-.64-2.56.64-5.44 3.2-6.08 23.36-7.04 52.16-3.52 72 8.64 1.92.96 2.88 4.16 1.28 6.4m.64-17.92C81.28 44.8 47.04 43.52 27.52 49.6c-3.2.96-6.4-.96-7.36-3.84-.96-3.2.96-6.4 3.84-7.36 22.72-6.72 60.16-5.44 83.84 8.64 2.88 1.6 3.84 5.44 2.24 8.32-1.6 2.24-5.44 3.2-8.32 1.6"
                          fill="#10BC4C"
                          fill-rule="evenodd"
                        />
                      </g>
                    </svg>
                  </a>
                </div>
                <button
                  class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
                >
                  unfollow
                </button>
              </div>
              <div
                class="flex justify-between items-center px-8 even:bg-gray-100"
              >
                <p>
                  YOYOO
                </p>
                <div
                  class="flex"
                >
                  <a
                    class="inline"
                    href="www.google.com"
                  >
                    <svg
                      height="40"
                      viewBox="0 0 112.195 112.195"
                      width="40"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M112.195 56.097c0 30.983-25.114 56.099-56.097 56.099C25.115 112.195 0 87.08 0 56.097 0 25.117 25.115 0 56.099 0c30.982 0 56.096 25.117 56.096 56.097z"
                        fill="#d31f27"
                      />
                      <path
                        d="M80.487 52.141c-6.203-1.849-8.412-2.874-8.412-4.922 0-3.416 4.937-4.702 5.497-4.79 3.139-.465 7.072 1.173 8.34 4.99l8.44-2.553c-2.65-8.385-10.487-12.099-18.099-10.975-7.788 1.146-12.782 6.786-12.782 13.327 0 8.8 8.244 11.332 14.43 13.181 6.509 1.944 8.401 2.637 8.401 5.023 0 2.016-1.196 3.895-4.374 4.735-6.32 1.664-14.586-.672-16.887-4.523-1.351-2.261-2.733-5.203-4.001-8.53-3.766-9.866-8.926-23.371-23.764-23.371-7.731 0-21.851 3.743-21.851 24.868 0 8.744 7.481 19.613 21.578 19.613 11.829 0 13.888-4.32 14.358-5.085L47.5 65.574c-.098.138-3.692 4.936-10.497 4.936-11.107 0-12.735-11.776-12.735-11.909 0-10.967 4.687-16.238 13.008-16.238 8.094 0 11.382 7.04 15.48 17.759 1.397 3.683 2.922 6.966 4.65 9.865 3.657 6.116 11.55 8.091 19.473 8.091 2.418 0 5.2.137 7.351-.432 7.093-1.878 10.905-6.178 10.905-12.224.002-9.204-8.283-11.378-14.648-13.281z"
                        fill="#fff"
                      />
                    </svg>
                  </a>
                </div>
                <button
                  class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
                >
                  unfollow
                </button>
              </div>
              <div
                class="flex justify-between items-center px-8 even:bg-gray-100"
              >
                <p>
                  Art Beckley
                </p>
                <div
                  class="flex"
                >
                  <a
                    class="inline"
                    href="www.google.com"
                  >
                    <svg
                      height="40"
                      viewBox="0 0 112.195 112.195"
                      width="40"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M112.195 56.097c0 30.983-25.114 56.099-56.097 56.099C25.115 112.195 0 87.08 0 56.097 0 25.117 25.115 0 56.099 0c30.982 0 56.096 25.117 56.096 56.097z"
                        fill="#d31f27"
                      />
                      <path
                        d="M80.487 52.141c-6.203-1.849-8.412-2.874-8.412-4.922 0-3.416 4.937-4.702 5.497-4.79 3.139-.465 7.072 1.173 8.34 4.99l8.44-2.553c-2.65-8.385-10.487-12.099-18.099-10.975-7.788 1.146-12.782 6.786-12.782 13.327 0 8.8 8.244 11.332 14.43 13.181 6.509 1.944 8.401 2.637 8.401 5.023 0 2.016-1.196 3.895-4.374 4.735-6.32 1.664-14.586-.672-16.887-4.523-1.351-2.261-2.733-5.203-4.001-8.53-3.766-9.866-8.926-23.371-23.764-23.371-7.731 0-21.851 3.743-21.851 24.868 0 8.744 7.481 19.613 21.578 19.613 11.829 0 13.888-4.32 14.358-5.085L47.5 65.574c-.098.138-3.692 4.936-10.497 4.936-11.107 0-12.735-11.776-12.735-11.909 0-10.967 4.687-16.238 13.008-16.238 8.094 0 11.382 7.04 15.48 17.759 1.397 3.683 2.922 6.966 4.65 9.865 3.657 6.116 11.55 8.091 19.473 8.091 2.418 0 5.2.137 7.351-.432 7.093-1.878 10.905-6.178 10.905-12.224.002-9.204-8.283-11.378-14.648-13.281z"
                        fill="#fff"
                      />
                    </svg>
                  </a>
                  <a
                    class="inline"
                    href="www.google.com"
                  >
                    <svg
                      height="40"
                      viewBox="0 0 128 128"
                      width="40"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <g
                        clip-rule="evenodd"
                      >
                        <path
                          d="M0 0h128v128H0z"
                          fill="none"
                        />
                        <path
                          d="M64 0C28.8 0 0 28.8 0 64s28.8 64 64 64 64-28.8 64-64S99.52 0 64 0m29.44 92.48c-1.28 1.92-3.52 2.56-5.44 1.28-15.04-9.28-33.92-11.2-56.32-6.08-2.24.64-4.16-.96-4.8-2.88-.64-2.24.96-4.16 2.88-4.8 24.32-5.44 45.44-3.2 62.08 7.04 2.24.96 2.56 3.52 1.6 5.44m7.68-17.6c-1.6 2.24-4.48 3.2-6.72 1.6-17.28-10.56-43.52-13.76-63.68-7.36-2.56.64-5.44-.64-6.08-3.2-.64-2.56.64-5.44 3.2-6.08 23.36-7.04 52.16-3.52 72 8.64 1.92.96 2.88 4.16 1.28 6.4m.64-17.92C81.28 44.8 47.04 43.52 27.52 49.6c-3.2.96-6.4-.96-7.36-3.84-.96-3.2.96-6.4 3.84-7.36 22.72-6.72 60.16-5.44 83.84 8.64 2.88 1.6 3.84 5.44 2.24 8.32-1.6 2.24-5.44 3.2-8.32 1.6"
                          fill="#10BC4C"
                          fill-rule="evenodd"
                        />
                      </g>
                    </svg>
                  </a>
                </div>
                <button
                  class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
                >
                  unfollow
                </button>
              </div>
            </div>
          </main>
          <footer
            class="border border-indigo-600 h-10 bg-blue-500"
          >
            <a
              href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
              rel="noopener noreferrer"
              target="_blank"
            >
              poweerf
              <span
                style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
              >
                <span
                  style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                >
                  <img
                    alt=""
                    aria-hidden="true"
                    src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNzIiIGhlaWdodD0iMTYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                    style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                  />
                </span>
                <img
                  alt="Vercel Logo"
                  data-nimg="intrinsic"
                  decoding="async"
                  src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                  style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                />
                <noscript />
              </span>
            </a>
          </footer>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container max-auto"
    >
      <div
        class="flex flex-col h-screen justify-between"
      >
        <main
          class="mb-auto"
        >
          <button>
            <svg
              class="h-6 w-6"
              fill="none"
              height="40"
              stroke="currentColor"
              viewBox="0 0 24 24"
              width="40"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M20.354 15.354A9 9 0 018.646 3.646 9.003 9.003 0 0012 21a9.003 9.003 0 008.354-5.646z"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
              />
            </svg>
          </button>
          <div
            class="flex justify-center items-center h-48 bg-red-300"
          >
            <form
              class="flex justify-center items-stretch h-10"
              novalidate=""
            >
              <input
                class="rounded mr-4 px-2 max-w-md"
                name="search"
                placeholder="Here goes the artist..."
                type="text"
              />
              <button
                class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
                type="submit"
              >
                Search
              </button>
            </form>
          </div>
          <div
            class="border border-indigo-100"
          >
            <div
              class="flex justify-between items-center px-8 even:bg-gray-100"
            >
              <p>
                Led Zeppeling
              </p>
              <div
                class="flex"
              >
                <a
                  class="inline"
                  href="www.google.com"
                >
                  <svg
                    height="40"
                    viewBox="0 0 128 128"
                    width="40"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      clip-rule="evenodd"
                    >
                      <path
                        d="M0 0h128v128H0z"
                        fill="none"
                      />
                      <path
                        d="M64 0C28.8 0 0 28.8 0 64s28.8 64 64 64 64-28.8 64-64S99.52 0 64 0m29.44 92.48c-1.28 1.92-3.52 2.56-5.44 1.28-15.04-9.28-33.92-11.2-56.32-6.08-2.24.64-4.16-.96-4.8-2.88-.64-2.24.96-4.16 2.88-4.8 24.32-5.44 45.44-3.2 62.08 7.04 2.24.96 2.56 3.52 1.6 5.44m7.68-17.6c-1.6 2.24-4.48 3.2-6.72 1.6-17.28-10.56-43.52-13.76-63.68-7.36-2.56.64-5.44-.64-6.08-3.2-.64-2.56.64-5.44 3.2-6.08 23.36-7.04 52.16-3.52 72 8.64 1.92.96 2.88 4.16 1.28 6.4m.64-17.92C81.28 44.8 47.04 43.52 27.52 49.6c-3.2.96-6.4-.96-7.36-3.84-.96-3.2.96-6.4 3.84-7.36 22.72-6.72 60.16-5.44 83.84 8.64 2.88 1.6 3.84 5.44 2.24 8.32-1.6 2.24-5.44 3.2-8.32 1.6"
                        fill="#10BC4C"
                        fill-rule="evenodd"
                      />
                    </g>
                  </svg>
                </a>
              </div>
              <button
                class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
              >
                unfollow
              </button>
            </div>
            <div
              class="flex justify-between items-center px-8 even:bg-gray-100"
            >
              <p>
                YOYOO
              </p>
              <div
                class="flex"
              >
                <a
                  class="inline"
                  href="www.google.com"
                >
                  <svg
                    height="40"
                    viewBox="0 0 112.195 112.195"
                    width="40"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M112.195 56.097c0 30.983-25.114 56.099-56.097 56.099C25.115 112.195 0 87.08 0 56.097 0 25.117 25.115 0 56.099 0c30.982 0 56.096 25.117 56.096 56.097z"
                      fill="#d31f27"
                    />
                    <path
                      d="M80.487 52.141c-6.203-1.849-8.412-2.874-8.412-4.922 0-3.416 4.937-4.702 5.497-4.79 3.139-.465 7.072 1.173 8.34 4.99l8.44-2.553c-2.65-8.385-10.487-12.099-18.099-10.975-7.788 1.146-12.782 6.786-12.782 13.327 0 8.8 8.244 11.332 14.43 13.181 6.509 1.944 8.401 2.637 8.401 5.023 0 2.016-1.196 3.895-4.374 4.735-6.32 1.664-14.586-.672-16.887-4.523-1.351-2.261-2.733-5.203-4.001-8.53-3.766-9.866-8.926-23.371-23.764-23.371-7.731 0-21.851 3.743-21.851 24.868 0 8.744 7.481 19.613 21.578 19.613 11.829 0 13.888-4.32 14.358-5.085L47.5 65.574c-.098.138-3.692 4.936-10.497 4.936-11.107 0-12.735-11.776-12.735-11.909 0-10.967 4.687-16.238 13.008-16.238 8.094 0 11.382 7.04 15.48 17.759 1.397 3.683 2.922 6.966 4.65 9.865 3.657 6.116 11.55 8.091 19.473 8.091 2.418 0 5.2.137 7.351-.432 7.093-1.878 10.905-6.178 10.905-12.224.002-9.204-8.283-11.378-14.648-13.281z"
                      fill="#fff"
                    />
                  </svg>
                </a>
              </div>
              <button
                class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
              >
                unfollow
              </button>
            </div>
            <div
              class="flex justify-between items-center px-8 even:bg-gray-100"
            >
              <p>
                Art Beckley
              </p>
              <div
                class="flex"
              >
                <a
                  class="inline"
                  href="www.google.com"
                >
                  <svg
                    height="40"
                    viewBox="0 0 112.195 112.195"
                    width="40"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M112.195 56.097c0 30.983-25.114 56.099-56.097 56.099C25.115 112.195 0 87.08 0 56.097 0 25.117 25.115 0 56.099 0c30.982 0 56.096 25.117 56.096 56.097z"
                      fill="#d31f27"
                    />
                    <path
                      d="M80.487 52.141c-6.203-1.849-8.412-2.874-8.412-4.922 0-3.416 4.937-4.702 5.497-4.79 3.139-.465 7.072 1.173 8.34 4.99l8.44-2.553c-2.65-8.385-10.487-12.099-18.099-10.975-7.788 1.146-12.782 6.786-12.782 13.327 0 8.8 8.244 11.332 14.43 13.181 6.509 1.944 8.401 2.637 8.401 5.023 0 2.016-1.196 3.895-4.374 4.735-6.32 1.664-14.586-.672-16.887-4.523-1.351-2.261-2.733-5.203-4.001-8.53-3.766-9.866-8.926-23.371-23.764-23.371-7.731 0-21.851 3.743-21.851 24.868 0 8.744 7.481 19.613 21.578 19.613 11.829 0 13.888-4.32 14.358-5.085L47.5 65.574c-.098.138-3.692 4.936-10.497 4.936-11.107 0-12.735-11.776-12.735-11.909 0-10.967 4.687-16.238 13.008-16.238 8.094 0 11.382 7.04 15.48 17.759 1.397 3.683 2.922 6.966 4.65 9.865 3.657 6.116 11.55 8.091 19.473 8.091 2.418 0 5.2.137 7.351-.432 7.093-1.878 10.905-6.178 10.905-12.224.002-9.204-8.283-11.378-14.648-13.281z"
                      fill="#fff"
                    />
                  </svg>
                </a>
                <a
                  class="inline"
                  href="www.google.com"
                >
                  <svg
                    height="40"
                    viewBox="0 0 128 128"
                    width="40"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      clip-rule="evenodd"
                    >
                      <path
                        d="M0 0h128v128H0z"
                        fill="none"
                      />
                      <path
                        d="M64 0C28.8 0 0 28.8 0 64s28.8 64 64 64 64-28.8 64-64S99.52 0 64 0m29.44 92.48c-1.28 1.92-3.52 2.56-5.44 1.28-15.04-9.28-33.92-11.2-56.32-6.08-2.24.64-4.16-.96-4.8-2.88-.64-2.24.96-4.16 2.88-4.8 24.32-5.44 45.44-3.2 62.08 7.04 2.24.96 2.56 3.52 1.6 5.44m7.68-17.6c-1.6 2.24-4.48 3.2-6.72 1.6-17.28-10.56-43.52-13.76-63.68-7.36-2.56.64-5.44-.64-6.08-3.2-.64-2.56.64-5.44 3.2-6.08 23.36-7.04 52.16-3.52 72 8.64 1.92.96 2.88 4.16 1.28 6.4m.64-17.92C81.28 44.8 47.04 43.52 27.52 49.6c-3.2.96-6.4-.96-7.36-3.84-.96-3.2.96-6.4 3.84-7.36 22.72-6.72 60.16-5.44 83.84 8.64 2.88 1.6 3.84 5.44 2.24 8.32-1.6 2.24-5.44 3.2-8.32 1.6"
                        fill="#10BC4C"
                        fill-rule="evenodd"
                      />
                    </g>
                  </svg>
                </a>
              </div>
              <button
                class="py-2 px-3 bg-indigo-800 text-white text-sm font-semibold rounded-md shadow focus:outline-none"
              >
                unfollow
              </button>
            </div>
          </div>
        </main>
        <footer
          class="border border-indigo-600 h-10 bg-blue-500"
        >
          <a
            href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
            rel="noopener noreferrer"
            target="_blank"
          >
            poweerf
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iNzIiIGhlaWdodD0iMTYiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="Vercel Logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </a>
        </footer>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
